packages/components/navigation/BuyTokens.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { Linking, TouchableOpacity } from "react-native";
+import { CreditCardIcon } from "react-native-heroicons/outline";
+
+import { primaryColor } from "../../utils/style/colors";
+import { fontBold16 } from "../../utils/style/fonts";
+import { BrandText } from "../BrandText";
+
+export const BuyTokens: React.FC = () => {
+  return (
+    <TouchableOpacity
+      style={{
+        justifyContent: "center",
+        alignItems: "center",
+        marginBottom: "0.5em",
```

Comment:
You should use the same space value as the other eleemnts in the Sidebar.
Like `padding.layout_x1`

Reply:
furthermost `em` is not supported by native